challenges/1-produto-simples/java/azuosz/produtoSimples.java

```diff
@@ -0,0 +1,26 @@
+import  java.util.Scanner;
+
+public class produtoSimples {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Digite um numero: ");
+        int  num1 = scanner.nextDouble();
+
+        System.out.print("Digite outro numero: " );
+        int num2 = scanner.nextDouble();
```

Comment:
Erro de compilação: Type mismatch: cannot convert from double to int.
```suggestion
        int num2 = scanner.nextInt();
```

Reply:
Outro ponto: classes no Java seguem o padrão [PascalCase](https://www.alura.com.br/artigos/convencoes-nomenclatura-camel-pascal-kebab-snake-case?#pascal-case), então sugiro renomear o nome da sua classe para `ProdutoSimples`.